conf/documentation.conf

```diff
@@ -1103,6 +1103,14 @@ The cost factor to apply to the password hashing if applicable.
 Currently only applies to bcrypt.
 EOT
 
+[advanced.disable_pf_domain_auth] 
+type=toggle
+options=enabled|disabled
+description=<<EOT
+Disables authentification on the domains defined in PacketFence.
+This will make it so the authentication only happens on the domain binded to the OS.
```

Comment:
Not sure this message is clear enough for a user that is not familiar with the FreeRADIUS/PacketFence/Winbind internals.
Maybe we should say something like 'disable Active-Directory domain configuration from within PacketFence'.